docs/quickstarts/insights-inventory-workspace-rbac/insights-inventory-workspace-rbac.yaml

```diff
@@ -14,25 +14,25 @@ spec:
   icon: data:image/svg+xml;base64,PCEtLSBHZW5lcmF0ZWQgYnkgSWNvTW9vbi5pbyAtLT4KPHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgd2lkdGg9IjUxMiIgaGVpZ2h0PSI1MTIiIHZpZXdCb3g9IjAgMCA1MTIgNTEyIj4KPHRpdGxlPjwvdGl0bGU+CjxnIGlkPSJpY29tb29uLWlnbm9yZSI+CjwvZz4KPHBhdGggZD0iTTQ0OCA2NHY0MTZoLTMzNmMtMjYuNTEzIDAtNDgtMjEuNDktNDgtNDhzMjEuNDg3LTQ4IDQ4LTQ4aDMwNHYtMzg0aC0zMjBjLTM1LjE5OSAwLTY0IDI4LjgtNjQgNjR2Mzg0YzAgMzUuMiAyOC44MDEgNjQgNjQgNjRoMzg0di00NDhoLTMyeiI+PC9wYXRoPgo8cGF0aCBkPSJNMTEyLjAyOCA0MTZ2MGMtMC4wMDkgMC4wMDEtMC4wMTkgMC0wLjAyOCAwLTguODM2IDAtMTYgNy4xNjMtMTYgMTZzNy4xNjQgMTYgMTYgMTZjMC4wMDkgMCAwLjAxOS0wLjAwMSAwLjAyOC0wLjAwMXYwLjAwMWgzMDMuOTQ1di0zMmgtMzAzLjk0NXoiPjwvcGF0aD4KPC9zdmc+Cg==
   prerequisites:
     - Be an Organization Administrator or have the User Access Administrator role
-    - Have Inventory Groups created
+    - Have Inventory Workspaces created
   description: |-
-    Learn how to manage access to your systems using Inventory groups
+    Learn how to manage access to your systems using Inventory workspaces
   introduction: |-
-    **Welcome to the quick start for managing user access using Inventory groups. Use this feature to enhance security within your organization.**
+    **Welcome to the quick start for managing user access using Inventory workspaces. Use this feature to enhance security within your organization.**
     
-    In this quick start, you will use the User Access application to configure access to different Inventory groups using custom role functionality and user access groups.
+    In this quick start, you will use the User Access application to configure access to different Inventory workspaces using custom role functionality and user access workspaces.
```

Comment:
So I just realized that some of the "group" words refer to user access groups, and if that's the case then the word should still be groups. So in this case it should say "to different inventory workspaces using custom role functionality and user access **groups**"

and inventory should be lower case